### PR TITLE
Fix 2185

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.3+ |
| Fixed tickets | #2185 |

Please, review the XML configuration because I don't know if it's correct.

### CODE REVIEW

File: cookbook/doctrine/custom_dql_functions.rst [hunk 1]
```diff
@@ -37,6 +39,8 @@ In Symfony, you can register your custom DQL functions as follows:
 
             <doctrine:config>
                 <doctrine:orm>
+                    <!-- don't forget to add this 'auto_mapping' option -->
+                    <doctrine:auto_mapping value="true" />
```
Comment (xabbuh) [inline cookbook/doctrine/custom_dql_functions.rst]: Afaik, this must be the `auto-mapping` attribute of the `orm` element.

Comment (stof) [inline cookbook/doctrine/custom_dql_functions.rst]: indeed

Comment (WouterJ) [inline cookbook/doctrine/custom_dql_functions.rst]: this also works btw

**Comment by timglabisch** [comment]
is this really necessary?

**Comment by WouterJ** [comment]
It is no longer needed and I vote for closing this PR.

What caused confusion (the reason an issue was opened), was the usage of the expanded notation in the config. In such a case, the `auto_mapping` setting should not be set to `doctrine.orm`, but to the config of the explicitely defined entity manager config (e.g. `doctrine.orm.entity_managers.default`). When you leave it in `doctrine.orm`, your entities won't be auto discovered.

Nowadays, this code example was updated to not use the expanded notation as it didn't make any sense to do so. So this can be closed as it's already covered by the `// ...` (the `auto_mapping` setting is included in the SE config).

**Pull request closed without merging**